tests/core/types-utils/test_types_utils.py

```diff
@@ -0,0 +1,135 @@
+import itertools
+
+import pytest
+
+from wasm import (
+    constants,
+)
+from wasm._utils.types import (
+    get_bit_size,
+    get_float_type,
+    get_integer_type,
+    is_float_type,
+    is_integer_type,
+)
+
+
+@pytest.mark.parametrize(
+    'get_X_type,bit_size',
+    itertools.product(
+        [get_float_type, get_integer_type],
+        (0, 31, 33, 63, 65),
+    ),
+)
+def test_get_X_type_invalid_bit_size(get_X_type, bit_size):
+    with pytest.raises(ValueError):
+        get_X_type(bit_size)
+
+
+@pytest.mark.parametrize(
+    'value,expected',
+    (
+        (32, constants.F32),
+        (64, constants.F64),
+    )
+)
+def test_get_float_type(value, expected):
+    actual = get_float_type(value)
+    # using `is` comparison here to ensure that we are using the same object,
+    # not just an equal string.
```

Comment:
I started with an `Enum` based approach but it proved untennable with how the library does things like indexing into type strings, etc so I chose this as a half-step.  Going to leave as-is since the test ends up testing the desired thing and should require minor updates when/if this moved to be an enum.

Reply:
Ok, no objection to leaving `'f32'` as a string. But the comment is wrong, or at least pointless:

```py
In [36]: s1 = 'str'

In [37]: s2 = 'str'

In [38]: s1 is s2
Out[38]: True
```
Might as well just make it an `==` test and drop the comment.